src/paths/dijkstra.c

```diff
@@ -320,6 +320,31 @@ igraph_error_t igraph_shortest_paths_dijkstra(const igraph_t *graph,
  * \function igraph_get_shortest_paths_dijkstra
  * \brief Weighted shortest paths from a vertex.
  *
+ * \ref igraph_get_shortest_paths_dijkstra() without a cutoff.
```

Comment:
Same comment as above, please keep the original docs, and add a reference to the cutoff version.